src/taipy/core/data/data_node.py

```diff
@@ -291,26 +293,20 @@ def lock_edition(self):
     def unlock_edit(self, at: datetime = None, job_id: JobId = None):
```

Comment:
since we will separate the last_edit_date from unlock_edit, we wouldn't need these 2 params anymore

Reply:
Absolutely, but for compatibility reasons, I kept the same signature, and I deprecated the two params.